test/vstest.console.UnitTests/Processors/RunSpecificTestsArgumentProcessorTests.cs

```diff
@@ -328,6 +371,113 @@ public void ExecutorExecuteShouldForValidSourcesAndValidSelectedTestsRunsTestsAn
             Assert.AreEqual(ArgumentProcessorResult.Success, argumentProcessorResult);
         }
 
+        [TestMethod]
+        public void RunTestsWithCommaSeperatedTests()
```

Comment:
nit: naming should follow `<UnitOfWork>Should<Behavior>When<Condition>`
nit: every test should have only one reason to fail

These two statements will help understand what feature is broken by just looking at the failed test case, adds to productivity.

Reply:
done